### ISSUE TITLE
Testcase 3D

### ISSUE THREAD

**Issue by bdc-ehealth** [hl7-be/lab#210]
@LotteAd 

I do not understand your testcase 3D (551), I can't find a test without units, and also the reference range has units... Do I misunderstand the test case?

**Comment by bdc-ehealth** [comment]
@LotteAd 

Testcase 3B (553) only contains the first option (high and low, and not high only and low only).

**Comment by bdc-ehealth** [comment]
@LotteAd,
Testcase 9A6 (557) conflicts with the decision we made in https://github.com/hl7-be/lab/issues/105#issuecomment-1068163733

**Comment by bert-cch** [comment]
@bdc-ehealth 

Is a response available for the above question?

**Comment by bdc-ehealth** [comment]
@bert-cch ,

Yes, Bert, I indicated the solution here: https://github.com/hl7-be/lab/issues/210#issuecomment-1669702413 .

You have to add 2 more observations, one with a reference range that only contains a high limit, and one that only contains a low limit.
